hack/test-go.sh

```diff
@@ -303,6 +309,9 @@ reportCoverageToCoveralls() {
   fi
 }
 
+# Start etcd
+kube::etcd::start
```

Comment:
And to be clear, I don't mean per-test-directory, I mean per-individual unit test... :/